ruby-challenges.rb

```diff
@@ -18,15 +27,59 @@
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Vermont', 'Washington'] 
 
 
+def states_list (hash)
+    hash.values.flatten.sort
+end
+#   p states_list(us_states)
+# Output: ["Arizona", "California", "Idaho", "Maine", "Nevada", "New Hampshire", "Oregon", "Vermont", "Washington"]
+
+# state_list(us_states)
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+
+    def initialize(model)
+        @model = model
+        @wheels = 2
+        @current_speed = 0
+    end
+
+    def get_info
+        "The #{model} bike has #{wheels} wheels and is going #{current_speed} mph."
```

Comment:
You may run into errors when calling this method if you do not call on the instance variables in the string interpolation.